public/index.html

```diff
@@ -23,8 +22,8 @@
                     </div>
                     <div class="col-md-4">
                         <div class="btn-group">
-                            <button type="button" ng-click="goToAdProduct()" class="btn btn-primary">Manage Products</button>
-                            <button type="button" ng-click="goToAdCheckout()" class="btn btn-success">Checkout</button>
+                            <button type="button" v-on:click="goToAdProduct" class="btn btn-primary">Manage Products</button>
```

Comment:
@mal90 this change is to keep it consistent with the previous angular change, but I think we can just use
```html
<a href="/products.html" class="btn btn-primary">Manage Products</a>
<a href="/checkout.html" class="btn btn-success">Checkout</a>
```

This way we can ignore all the vue/angular dependencies on this page. Otherwise, if you have future use for the frontend framework here we can just keep it like this.

Reply:
@viettrung9012  For now let's just v-on:click="goToAdProduct" . But i agree on your point , we do not need to have a method for navigation in that case . +1 for pointing that out .